client/utilities/sundayTheObserverSubscription.ts

```diff
@@ -8,3 +8,12 @@ export const isSundayTheObserverSubscription = (
 		productDetail.tier === 'Newspaper Digital Voucher - Observer'
 	);
 };
+
+export const isNotSundayTheObserverSubscription = (
+	productDetail: ProductDetail,
+): boolean => {
+	return (
+		productDetail.tier !== 'Newspaper Delivery - Observer' &&
+		productDetail.tier !== 'Newspaper Digital Voucher - Observer'
+	);
+};
```

Comment:
~please can we not have negative logic because I can't understand it~ 
please can we have positive logic because I can understand it better!
```suggestion
export const isSundayTheObserverSubscription = (
	productDetail: ProductDetail,
): boolean => {
	return (
		productDetail.tier === 'Newspaper Delivery - Observer' ||
		productDetail.tier === 'Newspaper Digital Voucher - Observer'
	);
};
```

Reply:
I moved these functions out into productResponse and renamed the non Observer one to `isGuardianProduct`